src/main/scala/gitbucket/core/controller/api/ApiRepositoryContentsControllerBase.scala

```diff
@@ -13,10 +13,13 @@ import scala.util.Using
 trait ApiRepositoryContentsControllerBase extends ControllerBase {
   self: ReferrerAuthenticator with WritableUsersAuthenticator with RepositoryCommitFileService =>
 
-  /*
-   * i. Get the README
-   * https://developer.github.com/v3/repos/contents/#get-the-readme
+  /**
+   * i. Get a repository README
+   * https://docs.github.com/en/rest/reference/repos#get-a-repository-readme
    */
+  get("/api/v3/repos/:owner/:repository/readme")(referrersOnly { repository =>
+    getContents(repository, "readme.md", params.getOrElse("ref", repository.repository.defaultBranch), true)
```

Comment:
README is not only `readme.md`.
https://github.com/gitbucket/gitbucket/blob/7544f64c65c8d59184ea65dab482c7ba459714c0/src/main/scala/gitbucket/core/controller/RepositoryViewerController.scala#L911-L913

Reply:
Oh, I completely forgot that...

Reply:
@takezoe Ah... So must fix this PR.

I wonder if there are more than two readme files with different extension, what will happen??

Reply:
I can't see any sorting or priority. The first one in the `readmeFiles`is rendered anyway?
In the below case, I verified that readme.pdf -> readme.md -> readme.txt

![image](https://user-images.githubusercontent.com/17608272/92308309-af85a580-efd7-11ea-958e-03d376a01ec6.png)

Reply:
It doesn't matter with this PR, I found a strange behavior. (maybe gitbucket-html5media-plugin's issue?)

![matryoshka](https://user-images.githubusercontent.com/17608272/92308473-b82aab80-efd8-11ea-88fe-cfa4373d137f.gif)

Reply:
@takezoe fixed PR at https://github.com/gitbucket/gitbucket/pull/2535/commits/54baa5d8bfa0c6f0e1416c1ee6ef3571557ee016. Now this api process readme with any extensions which can be rendered. 
At the same time `RepositoryViewerController.readmeFiles` moved to `RepositoryCommitFileService.readmeFiles` in order to user this List from `ApiRepositoryContentsControllerBase`.

Reply:
Thanks for fixing the pull request!

> I can't see any sorting or priority. The first one in the readmeFilesis rendered anyway?

Having explicit order would be better. We can consider it in another issue.